docs/parameters_desc.md

```diff
@@ -0,0 +1,189 @@
+  Only attributes (or groups of attributes) that have
+relative count greater than this value are returned.
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.minOIRatio</code></td>
+  <td><code>3.0</code></td>
+  <td>
+  Only attributes (or groups of attributes) that occur
+in the outliers <code>minInlierRatio</code> more times than in the inliers are returned.
+  </td>
+</tr>
+  <td><code>macrobase.analysis.zScore</code> and <code>macrobase.analysis.useZScore</code></td>
+  <td><code>3.0</code> and <code>false</code></td>
+  <td>
+  Specifies zScore threshold to distinguish the
+outliers from the inliers. The zScore is as defined by the detection mechanism's
+<code>getZScoreEquivalent</code> method.
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.targetPercentile</code> and <code>macrobase.analysis.usePercentile</code></td>
+  <td><code>0.99</code> and <code>true</code></td>
+  <td>
+  Specifies percentile threshold to
+distinguish the outliers from the inliers. If <code>targetPercentile</code> is set to 99%,
+then all points with scores greater than the score of the 99%th point are flagged
+as outliers.
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.mcd.alpha</code></td>
+  <td><code>0.5</code></td>
+  <td>
+  This is a <b>MCD-only</b> parameter. Controls the size of
+the cloud of points sampled from the entire dataset of points. In the first
+example described
+<a href="https://tr8dr.wordpress.com/2010/09/24/minimum-covariance-determination/">here</a>
+, <code>alphaMCD = 0.9</code>, for example.
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.mcd.stroppingDelta</code></td>
+  <td><code>0.001</code></td>
+  <td>
+  This is again a <b>MCD-only</b> parameter. Controls
+the stopping condition in the training phase of MCD.
+  </td>
+</tr>
+</table>
+
+
+The following are streaming-specific parameters, and make sense _only_ in
+streaming workloads.
+
+<table class="table">
+<tr><th>Name</th><th>Default</th><th>Meaning</th></tr>
+<tr>
+  <td><code>macrobase.analysis.streaming.inputReservoirSize</code></td>
+  <td><code>10,000</code></td>
+  <td>
+  Controls the number of points sampled from the
+input data stream. Whenever the outlier detection algorithm is trained, the tuples
+that are currently in the input reservoir are used. The larger this parameter,
+the more context the detection algorithms will have in the training phase. (and
+consequently, the longer the training phase will take)
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.analysis.streaming.scoreReservoirSize</code></td>
+  <td><code>10,000</code></td>
+  <td>
+  Controls the number of points sampled from the
+scored data stream. Used to estimate the threhold score when `usePercentile`
+(described above) is set to <code>true</code>. The larger this parameter, the more accurate
+the chosen threshold will be. (although more time would be spent to keep
+the reservoir sorted)
+  </td>
+</tr>
+  <td><code>macrobase.analysis.streaming.summaryUpdatePeriod</code></td>
```

Comment:
It is how often to decay the counts as well as restructure the tree.

Again, generally, I'd start with these periods, then the decay rate, then the remainder.